src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -122,6 +122,17 @@ class Telemetry : public PluginBase {
         float yaw_rad_s; /**< @brief Yaw angular velocity */
     };
 
+    /**
+     * @brief Ground speed type.
```

Comment:
Ground speed?

Reply:
I don't know from where this comes, i will check it